online-shop/src/components/ProductDetails.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { RouteComponentProps} from 'react-router-dom';
+import products from './productArray';
+import '../css/ProductDetails.css'
+import { makeStyles, createStyles, Theme } from '@material-ui/core/styles';
+import Grid from '@material-ui/core/Grid';
+import AddShoppingCartIcon from '@material-ui/icons/AddShoppingCart';
+import Button from '@material-ui/core/Button';
+
+type TParams = { id: string };
+
+const useStyles = makeStyles((theme: Theme) =>
```

Comment:
Stiluri in fisiere css. Aici s-ar putea sa nu poti, deoarece folosesti theme tho'.